lib/server/Server.js

```diff
@@ -22,6 +23,12 @@ var Server = {
         var appRoot = environmentConfig.appRoot;
         var debug = config.debug === true;
 
+        Deprecated.message(
+            "Brisket.ErrorViewMapping.create is deprecated. Instead use a plain object. " +
+            "Brisket.ErrorViewMapping will be removed in the next major release. ",
```

Comment:
should be:
`"Brisket.ErrorViewMapping will be removed in the next major release",`